lib/backend/dynamo/dynamodbbk.go

```diff
@@ -89,6 +89,8 @@ type Config struct {
 	// WriteTargetValue is the ratio of consumed write capacity to provisioned
 	// capacity. Required to be set if auto scaling is enabled.
 	WriteTargetValue float64 `json:"write_target_value,omitempty"`
+	// Tenant is value to be used as partition key.
```

Comment:
By default the dynamodb implementation enables the dynamo streams.
Take a deep dive do we really need the streams?